muxpi/dut.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
```

Comment:
As commented in the other script, it would be nice to have some introductory comment here with the purpose of the script.